1.9/usage/pods/examples.md

```diff
@@ -372,8 +721,8 @@ This pod declares a “web” endpoint that listens on port 80. <!-- Validated b
 }
 ```
 
-# Complete Pod 
-The following pod definition can serve as a reference to create more complicated pods. 
+# Complete Pod
```

Comment:
The simple and complete pod are the annotated examples. I put it in H1 so that it appears in the left hand TOC